vignettes/gallery.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Gallery"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Gallery}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+<br><br>
```

Comment:
Without this, the "Show X entries" row of the HTML output is squished between the package hex and table output.